parser/src/main/java/com/microsoft/gctoolkit/parser/unified/UnifiedLoggingTokens.java

```diff
@@ -29,7 +29,7 @@
     String TIME_MILLIS = "\\[\\d+ms\\]";
     String TIME_NANOS = "\\[\\d+ns\\]";
     String PID_TID = "\\[\\d+\\]";
-    String UNIFIED_LOG_LEVEL_BLOCK = "\\[(error|warning|info|debug|trace|develop)\\]";
+    String UNIFIED_LOG_LEVEL_BLOCK = "\\[(?:error|warning|info ?|debug|trace|develop)\\]";
```

Comment:
I still think there is an issue here. Let's say that the level is 'warning' then it will come out as 'info   ' (info plus three spaces). Or it could be debug plus two spaces, and so on. I think the safer pattern is:
```"\\[(?:error|warning|info|debug|trace|develop)\\s*\\]"```

Also, I thought you had added specific unit tests for this.